### DISCUSSION TITLE
hope that the account information of different PC computers can be synchronized

### DISCUSSION THREAD

**Discussion by lxmwwl** [Unlucky-Life/ankimon#150] [Q&A]

**Comment by CarlosJavierOrozco** [comment]
I hope that too! currently I use mega for that

**Comment by Unlucky-Life** [comment]
Im working on it currently and rewriting the addon basically. But you can sync your folder through GDrive or Mega etc. that would directly update the files to the online folder once you make any changes (by catching pokemon etc.)
